build-tools/automation/azure-pipelines.yaml

```diff
@@ -1197,11 +1198,18 @@ stages:
       parameters:
         UnsignedPkgPath: $(XA.Unsigned.Pkg)
 
+    - task: MicroBuildSigningPlugin@3
+      displayName: Install Signing Plugin
+      inputs:
+        signType: real
+        azureSubscription: MicroBuild Signing Task (DevDiv)
+      env:
+        TeamName: XamarinAndroid
+        SYSTEM_ACCESSTOKEN: $(system.accesstoken)
```

Comment:
case sensitivity for $(system.accesstoken)?  Might need to do $(System.AccessToken)